tests/implicit_import_test_two/proxy.cairo

```diff
@@ -0,0 +1,3 @@
+%lang starknet
```

Comment:
The directory name has to end with "test" for the pytest to catch it, for example `implicit_import_fp_test`. Also it misses the expected file.

Reply:
Good catch, thanks!